step-templates/Azure-Get-Publishing-Credentials.json

```diff
@@ -0,0 +1,56 @@
+{
+  "Id": "ActionTemplates-62",
+  "Name": "Get Azure Web App Publishing Credentials",
+  "Description": "Gets the publishing credentials for an Azure Web App. They are exported as variables from this step under the names PublishingUsername and PublishingPassword.", 
+  "ActionType": "Octopus.AzurePowerShell",
+  "Version": 8,
+  "CommunityActionTemplateId": null,
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.ScriptBody": "$creds = Invoke-AzureRmResourceAction -ResourceGroupName $ResourceGroup -ResourceType Microsoft.Web/sites/config `\r\n            -ResourceName $WebApp/publishingCredentials -Action list -ApiVersion 2015-08-01 -Force\r\n\r\nSet-OctopusVariable -name \"PublishingUsername\" -value $creds.Properties.PublishingUsername\r\nSet-OctopusVariable -name \"PublishingPassword\" -value $creds.Properties.PublishingPassword",
+    "Octopus.Action.Azure.AccountId": "#{AzureAccount}",
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "5a1d5a50-a950-42a4-85d6-25c2b9c45e91",
+      "Name": "ResourceGroup",
+      "Label": "Resource group",
+      "HelpText": "The name of the resource group that contains the web app for which publishing credentials are required.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "0f86965c-1f40-4778-8378-197ff4199330",
+      "Name": "WebApp",
+      "Label": "Web app",
+      "HelpText": "The name of the web app for which publishing credentials are required.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "572ee191-a035-47ab-aec3-ef805e690868",
+      "Name": "AzureAccount",
+      "Label": "Azure account",
+      "HelpText": null,
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    }
+  ],
```

Comment:
You need to add a `LastModifiedBy` property and set it to your :octocat: handle here